interop-future/jvm/src/main/scala/scalaz/zio/interop/FuturePlatformSpecific.scala

```diff
@@ -0,0 +1,27 @@
+package scalaz.zio.interop
+
+import scalaz.zio.{ ExitResult, FiberFailure, IO, RTS }
+import scala.concurrent.{ Future, Promise }
+import scala.util.{ Failure, Success }
+
+protected[interop] trait FuturePlatformSpecific {
+  implicit class RTSOps(private val rts: RTS) {
```

Comment:
wouldn't this create a instance of `RTSOps` on every `unsafeToFuture` call? Could it be made extending `AnyVal`?

Reply:
Unfortunately not, because it’s in a trait. Happy to juggle this around if you have an idea.

OTOH, if you’re running unsafeToFuture, you already are allocating monstrously... :-)

Reply:
You could split implicit class into AnyVal in object + implicit def in trait if you wanted to.

Reply:
Ahh, thanks!